backend/service/chaos/experimentation/experimentstore/experiment_store_test.go

```diff
@@ -85,13 +103,16 @@ func TestCreateExperiments(t *testing.T) {
 
 			es := &experimentStore{db: db}
 			defer es.Close()
-
-			expected := mock.ExpectExec(regexp.QuoteMeta(test.sql))
-			if test.err != nil {
-				expected.WillReturnError(test.err)
-			} else {
-				expected.WithArgs(sqlmock.AnyArg(), test.args[1]).WillReturnResult(sqlmock.NewResult(1, 1))
+			mock.ExpectBegin()
+			for _, query := range test.queries {
+				expected := mock.ExpectExec(regexp.QuoteMeta(query.sql))
+				if test.err != nil {
+					expected.WillReturnError(test.err)
```

Comment:
is this ever executed? We don't have an "erroring" test case now, no? Can we get rid of it? Or maybe we need to modify our test case so that some of them are expected to fail (return an error) again?